### DISCUSSION TITLE
Change ComboBox color

### DISCUSSION THREAD

**Discussion by md-rifatkhan** [bdlukaa/fluent_ui#1089] [Q&A] [answered]
How can i change combo box color?

**Comment by bdlukaa** [comment, marked as answer]
To change the popup color, use the `ComboBox.popupColor` property. If you want to change the `ComboBox` button color, you can change the button theme above it using the [`ButtonTheme`](https://pub.dev/documentation/fluent_ui/latest/fluent_ui/ButtonTheme-class.html)
